app/views/dashboards/show.html.erb

```diff
@@ -27,19 +27,19 @@
     </div>
 
     <div class="col-md">
-      <h2><%= t ".my friends" %></h2>
+      <h2><%= t ".followed users" %></h2>
```

Comment:
Maybe use "Following" instead of "Followed users" for better clarity. It aligns with the common terminology users are familiar with, like "Following" and "Followers," which may feel more intuitive. Just a thought.

Reply:
I agree with you. Wording was changed from `followed users` to `followings`. I was also thinking about it. `followed users` is more understandable, but `followings` keeps sufficient understandability and makes terminology shorter and more visually appealing.